client/go/build.gradle

```diff
@@ -48,3 +55,17 @@ build {
         ant.chmod(file: "$project.ext.binDir" + '/dogma.windows-amd64.exe', perm: '644')
     }
 }
+
+FileTree goSources = fileTree("${project.projectDir}").include('**/*.go').exclude('.gogradle/**')
+
+tasks.whenTaskAdded { task ->
+    def taskName = task.name
+    if (taskName.startsWith('build')) {
```

Comment:
But it should not intercept the task 'build', right?

Reply:
Oops~